### ISSUE TITLE
TZ environment variable not taken into account into zabbix-server, zabbix-proxy & zabbix-agent RHEL images

### ISSUE THREAD

**Issue by grunlab** [zabbix/zabbix-docker#953]
TZ environment variable is not taken into account into zabbix-server, zabbix-proxy & zabbix-agent RHEL images.

Impact is mainly at zabbix-server level --> the macro {EVENT.TIME} used into email notifications is not displayed in the correct timezone.

I've tried with Ubuntu images --> no issue, TZ environment variable is correctly taken into account. The issue looks specific to RHEL images.
I think the issue started when I switched from centos images to rhel images.

For zabbix-console RHEL image: no issue, TZ and PHP_TZ environment variables are correctly taken into account.

**Comment by grunlab** [comment]
Certainly because of this: https://access.redhat.com/solutions/5616681

**Comment by grunlab** [comment]
Workaround --> rebuild the image with :

```
    rpm --erase --nodeps tzdata && \
    microdnf --disablerepo='*' --enablerepo=ubi8-baseos install -y --setopt=tsflags=nodocs tzdata && \
```

Nota: I'm getting the packages from a local artifactory on LAN (no direct access to internet)

**Comment by dotneft** [comment]
Actually it must be fixed on RHEL side, but the solution is opened already 2 years. So, I think, it is better to fix on Zabbix side :-) Will be fixed soon.

**Comment by grunlab** [comment]
FYI, as we are also using redhat ubi images for other topics, i will also open a case on redhat side to have this issue solved at ubi based image level (which is effectively logic)

**Comment by grunlab** [comment]
Extract of https://access.redhat.com/solutions/5616681
```
This removal of /usr/share/zoneinfo is intentional in Red Hat Base Universal Minimal Images.
```

So I think we have no choice --> tzdata must be reinstalled on top of the UBI based image if we want to use TZ environment variable ...

**Comment by dotneft** [comment]
Fixed!

**Comment by grunlab** [comment]
Great, thank you :-)

The fix will be included starting 6.0.3-1 ? (not yet available : https://catalog.redhat.com/software/containers/zabbix/zabbix-server-mysql-60/622f43bb4a14c05796114d0e?tag=latest)

**Comment by dotneft** [comment]
Looks like RedHat changed image publishing method again... Please wait.
